### PR TITLE
Add contributor list script

### PR DESCRIPTION
**Pull request by fhemberger**
So far I've been using https://github.com/bengl/contribution-report for getting the list of contributors. Unfortunately, the names of the contributors are missing and require a separate API call each. Enter GitHub GraphQL API: Here all information can be pulled with one API call.

What's missing:
- Get all existing collaborators for the repo (couldn't find out the right query)
- Implement pagination
- Add nice markdown-ready output

### CODE REVIEW

**Comment by refack** [comment]
Just cleaning up the bits

File: scripts/contributor-list/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+const path = require('path')
+
+const GithubGraphQLApi = require('node-github-graphql')
+const github = new GithubGraphQLApi({
+  Promise,
+  token: process.env.GITHUB_API_TOKEN,
+  userAgent: 'nodejs.org-contributor'
+})
+
+const args = process.argv.splice(2)
```
Comment (lpinca) [inline scripts/contributor-list/index.js]: Nit: I'd use `slice` to not mutate `process.argv` :)

**Comment by fhemberger** [comment]
@refack Could you please help with the second GraphQL query (and/or pagination)? I'd love to finish this.

**Comment by fhemberger** [comment]
:100: thanks to @DerTieran for his tremendous help!
@lpinca Please take a look, I think it's good to merge.

**Comment by lpinca** [comment]
LGMT but I don't know much about GraphQL.

**Comment by fhemberger** [comment]
Let's compare the result of the old script with this one on the next onboarding. If they match, we're good to go.

**Pull request merged**